docs/reference/installation.rst

```diff
@@ -1,13 +1,22 @@
 Installation
 ============
 
-To begin, add the dependent bundles:
+Retrieve the bundle with composer:
 
 .. code-block:: bash
 
     composer require sonata-project/notification-bundle
+
+
+Or if you wish to use doctrine backend, execute ``composer require sonata-project/notification-orm-pack``.
+Symfony Flex will download recipes and install all necessary configuration and entity class for notification messages.
```

Comment:
entity classes? entities?

Reply:
there is only one entity class `App\Entity\SonataNotificationMessage`

Reply:
:thinking: … maybe this then: "Symfony Flew will download recipes and install all necessary configuration files, and an entity class for notification messages" ?

Reply:
Done. Thank you for fixing me )